src/app/story/storyrow/storyrow.component.html

```diff
@@ -65,3 +65,11 @@
     </div>
   </div>
 </div>
+<div class="node-siblings-column story-row-column total-time-column" *ngIf="first">
```

Comment:
This code isn't tested as far as I can tell. You need to add a unit test to make sure the timeline shows up on the first row.